File: Primeira lista pt2/Exercicio04.java
public class Exercicio04 {
    public static void main(String[] args) {

        double nigeria = 195.9;
        double china = 1393.0;

        double crescimentoNigeria = 0.038;
        double crescimentoChina = 0.0037;

        int anos = 0;

        while (nigeria <= china) {

            nigeria = nigeria * (1 + crescimentoNigeria);
            china = china * (1 + crescimentoChina);

            anos++;

            System.out.printf(
                "Ano %d - Nigéria: %.2f milhões | China: %.2f milhões%n",
                anos, nigeria, china
            );
        }

        System.out.println("Quantidade de anos necessários: " + anos);
    }
}

File: lista pt2/Exercicio04.js
let nigeria = 195.9;
let china = 1393.0;

let crescimentoNigeria = 0.038;
let crescimentoChina = 0.0037;

let anos = 0;

while (nigeria <= china) {

    nigeria = nigeria + (nigeria * crescimentoNigeria);
    china = china + (china * crescimentoChina);

    anos++;

    console.log(
        "Ano " + anos +
        " - Nigéria: " + nigeria.toFixed(2) +
        " milhões | China: " + china.toFixed(2) +
        " milhões"
    );
}

console.log("Quantidade de anos necessários: " + anos);